PullUpController/PullUpController.swift

```diff
@@ -59,6 +59,27 @@ open class PullUpController: UIViewController {
         return []
     }
     
+    /**
+     Determines if this pull up controller can go fullscreen
+     */
+    open var pullUpControllerCanGoFullscreen: Bool {
+        return true;
+    }
+    
+    /**
+     Determines if this pull up controller can fully hide below the screen
+     */
+    open var pullUpControllerCanHide: Bool {
+        return true;
+    }
+    
```

Comment:
`pullUpControllerCanGoFullscreen` and `pullUpControllerCanHide` don't seems to work or, maybe, I misunderstood what they should do.
You should remove the semicolon anyway.